tools/cold-backup/run.sh

```diff
@@ -0,0 +1,252 @@
+#!/usr/bin/env bash
+set -Eeuo pipefail
+#set -x #debug
+__TMP="/tmp/strimzi-backup" && mkdir -p $__TMP
+__HOME="" && pushd "$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")" >/dev/null \
+    && { __HOME=$PWD; popd >/dev/null; }
+__MPOD="maintenance"
+
+SOURCE_NS="kafka"
+TARGET_NS="kafka"
+CLUSTER_NAME="my-cluster"
+BACKUP_NAME="$SOURCE_NS-$(date +%Y%m%d)"
+BACKUP_HOME="/tmp"
+
+# PVC size must match with the corresponding CR value
+# PVC class must be empty when not available
+# specify number of JBOD volumes or zero
+ZOO_REPLICAS="3"
```

Comment:
For this and the `KAFKA_REPLICAS` can't we obtain the number from the CR using `kubectl get ... -o jsonpath ...` ?

Reply:
Sure. Will do that change.